### DISCUSSION TITLE
What mechanism(s) should we use for community engagement?

### DISCUSSION THREAD

**Discussion by reece** [HGVSnomenclature/hgvs-nomenclature#55] [Other discussion topics]
At the Aug 7 HVNC meeting, we started a discussion regarding the best ways to engage the community and document frequently asked questions. This thread is to solicit ideas. 

The most useful comments will discuss use cases (e.g., I want to ask someone about the correct use of repeats) and possible solutions (e.g., Slack, Discord, etc.)

The historical and current mechanisms are:

- facebook (deprecated)
- email address (to Johan, not an email group)
- presentations at meetings

**Comment by reece** [comment]
Ideas:

- For bugs and errors, submission to GitHub issues
- For questions/discussion, submission to GitHub discussions, Slack, or Google Group email
- For repeated questions, curation of FAQs by HVNC in website

**Reply by ifokkema** [reply]
I like the idea of a Google Group email for questions; it can also be a public archive that can be linked to in case people have similar questions. If discussion is needed, we can then always move to GitHub but include a link to it from the Google Group.

**Comment by reece** [comment]
New group created at https://groups.google.com/g/hgvs-nomenclature . Any email address works.
